### ISSUE TITLE
[BUG] List devices reports bogus OpenCL devices

### ISSUE THREAD

**Issue by RainbowMiner** [sp-hash/TeamBlackMiner#13]
On a Nvidia rig with one onboard Intel GPU, `tbminer --list-devices` reports bogus OpenCL values:

```
Available Cuda devices
-----------------------

Device Number: 0
    Device name: NVIDIA GeForce GTX 1070
    Memory clock rate (KHz): 4004000
    Memory bus width (bits): 256
    Peak memory bandwidth (GB/s): 256.256000

Device Number: 1
    Device name: NVIDIA GeForce GTX 1070
    Memory clock rate (KHz): 4004000
    Memory bus width (bits): 256
    Peak memory bandwidth (GB/s): 256.256000

Device Number: 2
    Device name: NVIDIA GeForce GTX 1070
    Memory clock rate (KHz): 4004000
    Memory bus width (bits): 256
    Peak memory bandwidth (GB/s): 256.256000

Device Number: 3
    Device name: NVIDIA GeForce GTX 1070
    Memory clock rate (KHz): 4004000
    Memory bus width (bits): 256
    Peak memory bandwidth (GB/s): 256.256000

Device Number: 4
    Device name: NVIDIA GeForce GTX 1070
    Memory clock rate (KHz): 4004000
    Memory bus width (bits): 256
    Peak memory bandwidth (GB/s): 256.256000

Device Number: 5
    Device name: NVIDIA GeForce GTX 1070
    Memory clock rate (KHz): 4004000
    Memory bus width (bits): 256
    Peak memory bandwidth (GB/s): 256.256000

Device Number: 6
    Device name: NVIDIA GeForce RTX 3070
    Memory clock rate (KHz): 7001000
    Memory bus width (bits): 256
    Peak memory bandwidth (GB/s): 448.064000


Available OpenCL devices
-------------------------

Device Number: 0
    Device name: Intel(R) HD Graphics 610
    Device max configured clock rate (MHz): 1050
    Max size of memory allocation in bytes: 1694636032
    Size of global memory cache in bytes: 262144

Device Number: 1
    Device name: Intel(R) HD Graphics 610
    Device max configured clock rate (MHz): 1050
    Max size of memory allocation in bytes: 1694636032
    Size of global memory cache in bytes: 262144

Device Number: 2
    Device name: Intel(R) HD Graphics 610
    Device max configured clock rate (MHz): 1050
    Max size of memory allocation in bytes: 1694636032
    Size of global memory cache in bytes: 262144

Device Number: 3
    Device name: Intel(R) HD Graphics 610
    Device max configured clock rate (MHz): 1050
    Max size of memory allocation in bytes: 1694636032
    Size of global memory cache in bytes: 262144

Device Number: 4
    Device name: Intel(R) HD Graphics 610
    Device max configured clock rate (MHz): 1050
    Max size of memory allocation in bytes: 1694636032
    Size of global memory cache in bytes: 262144

Device Number: 5
    Device name: Intel(R) HD Graphics 610
    Device max configured clock rate (MHz): 1050
    Max size of memory allocation in bytes: 1694636032
    Size of global memory cache in bytes: 262144

Device Number: 6
    Device name: Intel(R) HD Graphics 610
    Device max configured clock rate (MHz): 1050
    Max size of memory allocation in bytes: 1694636032
    Size of global memory cache in bytes: 262144
```

**Comment by sp-hash** [comment]
fixed in v1.02

**Comment by RainbowMiner** [comment]
Sorry, it's not solved with v1.02, yet. Just tried:
```
Available Cuda devices
-----------------------

Device Number: 0
    Device name: NVIDIA GeForce GTX 1070
    Memory clock rate (KHz): 4004000
    Memory bus width (bits): 256
    Peak memory bandwidth (GB/s): 256.256000

Device Number: 1
    Device name: NVIDIA GeForce GTX 1070
    Memory clock rate (KHz): 4004000
    Memory bus width (bits): 256
    Peak memory bandwidth (GB/s): 256.256000

Device Number: 2
    Device name: NVIDIA GeForce GTX 1070
    Memory clock rate (KHz): 4004000
    Memory bus width (bits): 256
    Peak memory bandwidth (GB/s): 256.256000

Device Number: 3
    Device name: NVIDIA GeForce GTX 1070
    Memory clock rate (KHz): 4004000
    Memory bus width (bits): 256
    Peak memory bandwidth (GB/s): 256.256000

Device Number: 4
    Device name: NVIDIA GeForce GTX 1070
    Memory clock rate (KHz): 4004000
    Memory bus width (bits): 256
    Peak memory bandwidth (GB/s): 256.256000

Device Number: 5
    Device name: NVIDIA GeForce GTX 1070
    Memory clock rate (KHz): 4004000
    Memory bus width (bits): 256
    Peak memory bandwidth (GB/s): 256.256000

Device Number: 6
    Device name: NVIDIA GeForce RTX 3070
    Memory clock rate (KHz): 7001000
    Memory bus width (bits): 256
    Peak memory bandwidth (GB/s): 448.064000


Available OpenCL devices
-------------------------

Device Number: 0
    Device name: Intel(R) HD Graphics 610
    Device max configured clock rate (MHz): 1050
    Max size of memory allocation in bytes: 1694636032
    Size of global memory cache in bytes: 262144

Device Number: 1
    Device name: Intel(R) HD Graphics 610
    Device max configured clock rate (MHz): 1050
    Max size of memory allocation in bytes: 1694636032
    Size of global memory cache in bytes: 262144

Device Number: 2
    Device name: Intel(R) HD Graphics 610
    Device max configured clock rate (MHz): 1050
    Max size of memory allocation in bytes: 1694636032
    Size of global memory cache in bytes: 262144

Device Number: 3
    Device name: Intel(R) HD Graphics 610
    Device max configured clock rate (MHz): 1050
    Max size of memory allocation in bytes: 1694636032
    Size of global memory cache in bytes: 262144

Device Number: 4
    Device name: Intel(R) HD Graphics 610
    Device max configured clock rate (MHz): 1050
    Max size of memory allocation in bytes: 1694636032
    Size of global memory cache in bytes: 262144

Device Number: 5
    Device name: Intel(R) HD Graphics 610
    Device max configured clock rate (MHz): 1050
    Max size of memory allocation in bytes: 1694636032
    Size of global memory cache in bytes: 262144

Device Number: 6
    Device name: Intel(R) HD Graphics 610
    Device max configured clock rate (MHz): 1050
    Max size of memory allocation in bytes: 1694636032
    Size of global memory cache in bytes: 262144
```
